packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -3,12 +3,12 @@ import { useCopyToClipboard } from 'react-use'
 import { Popover } from '@headlessui/react'
 import {
   ChevronDownIcon,
-  ExternalLinkIcon,
-  LogoutIcon,
+  ArrowTopRightOnSquareIcon,
+  ArrowLeftOnRectangleIcon,
   DocumentTextIcon,
   DocumentDuplicateIcon,
   CogIcon
-} from '@heroicons/react/outline'
+} from '@heroicons/react/24/outline'
 import { JsonRpcProvider } from '@ethersproject/providers'
```

Comment:
Previous cog-icon design has been updated. We should now be using `Cog6ToothIcon`

Reply:
Yes please change it @chiefmasta

Reply:
@fionnachan @dewanshparashar Done ✅